File: mjkim103301/baseball/src/test/java/com/hanwha/player/UserTest.java
package com.hanwha.player;

import static org.assertj.core.api.Assertions.assertThatThrownBy;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

class UserTest {

    @DisplayName("잘못된 사용자 입력 테스트: 중복 숫자 존재")
    @Test
    void userInputTest() {
        User user = new User(113);
        assertThatThrownBy(user::checkValidInput)
            .isInstanceOf(IllegalArgumentException.class);
    }
}

File: mjkim103301/pay-separately/src/test/kotlin/org/example/pay/model/UserTest.kt
package org.example.pay.model

import org.assertj.core.api.SoftAssertions.assertSoftly
import org.junit.jupiter.api.Assertions.*
import kotlin.test.Test

class UserTest{

    @Test
    fun `사용자 조회 테스트`(){
        val user = listOf(
            User(1, "Alice"),
            User(2, "Bob"),
            User(3, "Charlie"),
            User(4, "Dave"),
            User(5, "Minji")
        )
        assertSoftly {
            for (user in user) {
                println(user.toString())
            }
        }
    }
}
